elements/pfe-markdown/test/pfe-markdown.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+
+// Import the element we're testing.
+import { PfeMarkdown } from '@patternfly/pfe-markdown';
+
+// One element, defined here, is used
+// in multiple tests. It's torn down and recreated each time.
+const TEMPLATES = {
+  'original': html`<pfe-markdown>
+    <div pfe-markdown-container># pfe-markdown</div>
+  </pfe-markdown>`,
+  'dynamic': html`<pfe-markdown>
+    <div pfe-markdown-container></div>
+  </pfe-markdown>`,
+  'added': html`<pfe-markdown>
+    <div pfe-markdown-container># pfe-markdown</div>
+  </pfe-markdown>`,
+  'removed': html`<pfe-markdown>
+    <div pfe-markdown-container># pfe-markdown</div>
+  </pfe-markdown>`,
+};
+
+describe('<pfe-markdown>', function() {
+  it('should upgrade pfe-markdown', async function() {
+    const pfeAccordion = await createFixture<PfeMarkdown>(html`<pfe-markdown></pfe-markdown>`);
+    expect(pfeAccordion, 'pfe-markdown should be an instance of PfeMarkdown')
```

Comment:
Rename variable `pfeAccordion` to `element`

```Suggestion
    const element = await createFixture<PfeMarkdown>(html`<pfe-markdown></pfe-markdown>`);
    expect(element, 'pfe-markdown should be an instance of PfeMarkdown')
```

Reply:
Yeah I'm all for this